src/safeds/ml/nn/layers/_forward_layer.py

```diff
@@ -21,18 +21,25 @@ class ForwardLayer(Layer):
     neuron_count:
         The number of neurons in this layer
 
+    activation_function:
+        The activation function used in the forward layer
+
     Raises
     ------
     OutOfBoundsError
         If input_size < 1
         If output_size < 1
+
+    ValueError
+        If the given activation function does not exist
     """
 
-    def __init__(self, neuron_count: int):
+    def __init__(self, neuron_count: int, activation_function: str = "notset"):
         _check_bounds("neuron_count", neuron_count, lower_bound=_ClosedBound(1))
 
         self._input_size: int | None = None
         self._output_size = neuron_count
+        self._activation_function: str = activation_function
```

Comment:
Include self._activation_function in __hash__, __sizeof__ and __eq__

Reply:
added